src/Common/Auth/OpenIDConnect/Repositories/ScopeRepository.php

```diff
@@ -457,6 +573,7 @@ public function getOidcSupportedScopes(): array
         return $this->oidcScopes();
     }
 
+    // TODO: @bradymiller is this used anywhere anymore?
```

Comment:
not seeing this function used anywhere in master or in this PR, so can remove it

Reply:
Removed.